reference/configuration/security.rst

```diff
@@ -900,6 +900,54 @@ multiple firewalls, the "context" could actually be shared:
     ignored and you won't be able to authenticate on multiple firewalls at the
     same time.
 
+stateless
+~~~~~~~~~
+
+Firewalls can configure a ``stateless`` boolean option in order to declare that the session mustn't be used when authenticating user:
```

Comment:
Perhaps detail that in dev env, it will warn the developer ?

Reply:
Before https://github.com/symfony/symfony-docs/issues/17581, there is no triggered warning, because warning only on request `_stateless` attribute